orbstation/code/_global/global_procs.dm

```diff
@@ -0,0 +1,14 @@
+// Initializes the global language list. Used in a couple different places. Named this way for future-proofing.
+/proc/orb_init_languages()
+	if(GLOB.all_languages.len)
+		return
+	for(var/language in subtypesof(/datum/language))
+		var/datum/language/language_type = language
+		if(!initial(language_type.key))
```

Comment:
As commented you could do this:
```suggestion
	for(var/datum/language/language_type as anything in subtypesof(/datum/language))
		if(!initial(language_type.key))
```
Optional though if you think the existing code is clearer

Reply:
I missed the "as anything" earlier. That makes so much sense.